Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -245,22 +245,11 @@ open class ChatMessageListVC: _ViewController,
             // Because we use an inverted table view, we need to avoid animations since they look odd.
             UIView.performWithoutAnimation {
                 listView.updateMessages(with: changes) { [weak self] in
-                    guard let self = self else {
-                        completion?()
-                        return
-                    }
-
                     if let newMessageInserted = changes.first(where: { $0.isInsertion && $0.indexPath.row == 0 })?.item {
-                        if self.dataSource?.numberOfMessages(in: self) ?? 0 > 1 {
-                            let previousMessageIndexPath = IndexPath(row: 1, section: 0)
-                            self.listView.reloadRows(at: [previousMessageIndexPath], with: .none)
-                        }
```

Comment:
We should manually reload the previous cell for any insert and delete (except maybe when the new message is sent by the current user). Imagine a chat participant sending 2 messages and then hard deleting the last one. In that case, we should reload his previous message to show avatar etc.

Reply:
I see 👍 I'll apply that fix tomorrow and test it 🙂

Reply:
This is not needed, it is working fine without any change 🙂

Reply:
How so? It works fine only for the new message case because previous message stops being a channel preview and receives and update. But it removal happens in the middle of the list, there’s no other way but to manually reload

Reply:
I see what you mean. But we didn't have this working previously, right? Should we create an issue for this? Just want to try and avoid so many changes. This is a small edge case anyway

Reply:
We were handling this case, not sure when we stopped doing that

Reply:
That's a regression introduced several releases ago. Agreed to have a ticket and fix it later because this can potentially lead to more crashes which would be more disappointing for customers then having this edge-case uncovered for 2 more weeks.